src/modules/squeeze/squeeze.controller.ts

```diff
@@ -0,0 +1,43 @@
+import { Body, Controller, HttpCode, Put } from '@nestjs/common';
+import { SqueezeService } from './squeeze.service';
+import { UpdateSqueezeDto } from './dto/update-squeeze.dto';
+import { ApiResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { skipAuth } from '../../helpers/skipAuth';
+
+@ApiTags('Squeeze')
+@Controller('squeeze')
+export class SqueezeController {
+  constructor(private readonly squeezeService: SqueezeService) {}
+
+  @skipAuth()
```

Comment:
Is this route protected? if yes remove this decorator

Reply:
it's not protected.